packages/odyssey-react-mui/src/theme/createOdysseyMuiTheme.ts

```diff
@@ -23,28 +23,29 @@ import "./components.types";
 import "./mixins.types";
 import "./palette.types";
 import "./typography.types";
+import { type ContrastMode } from "../useContrastMode";
 
 export type DesignTokens = typeof Tokens;
 export type DesignTokensOverride = Partial<typeof Tokens>;
 
 export const createOdysseyMuiTheme = ({
-  contrastMode = "highContrast",
+  contrastMode = "lowContrast",
```

Comment:
This default value should be exported from `useContrastMode` and also imported in `OdysseyThemeProvider` as a default there because we have that `||` check.